app/Actions/ValidateConfiguration.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Actions;
+
+use App\ConsoleWriter;
+
+class ValidateConfiguration
+{
+    protected $consoleWriter;
+
+    public function __construct(ConsoleWriter $consoleWriter)
+    {
+        $this->consoleWriter = $consoleWriter;
+    }
+
+    public function __invoke()
+    {
+        $this->consoleWriter->logStep('Validating configuration');
+
+        config(['lambo.store.frontend' => $this->getFrontendConfiguration()]);
+
+        $this->checkTeamsConfiguration();
+
+        $this->consoleWriter->verbose()->success('Configuration is valid.');
+    }
+
+    protected function getFrontendConfiguration(): string
```

Comment:
I think I'd build this more like this:

```php
        $inertia = config('lambo.store.inertia');
        $livewire = config('lambo.store.livewire');

        if ($inertia && $livewire) {
            return $this->chooseBetweenFrontends();
        }

        if (! $inertia && ! $livewire) {
            return 'none';
        }

        return $inertia : 'inertia' : 'livewire'
    }
```

and then the `chooseBetweenFrontends` method would be basically the rest of this method, where you make them choose.

Reply:
Can I do this in beta3?